src/common/class.wpes-wp-post-field-builder.php

```diff
@@ -534,6 +534,127 @@ public function get_mappings( $args = array() ) {
 						),
 					),
 				),
+				'attachment' => array(
```

Comment:
I want us to move to having all images in there. We'll probably add in remote oEmbed calls to get meta info when indexing images hosted elsewhere. Not sure when we'll get there, but I am 100% sure we need to get there.